pallets/connectors/src/lib.rs

```diff
@@ -196,6 +214,8 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
+		/// Failed to map the asset to its u128 representation
```

Comment:
```suggestion
		/// Failed to map the asset to its Connector's General Index representation
```

I guess we would be better off being more specific?